app/main.py

```diff
@@ -1,3 +1,52 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.car import Car
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        data = json.load(file)
+    Car.fuel_price = data["FUEL_PRICE"]
+    customers_list = [0] * len(data["customers"])
```

Comment:
I can't really see what's the point of doing this instead of creating an empty list and appending customer objects to it. Can you explain?

Reply:
brain lag)